options.go

```diff
@@ -37,6 +37,16 @@ type configuration struct {
 	App string
 }
 
+// HashRingConfiguration is a configuration struct that can be passed to the
+// Ringpop constructor to customize hash ring options.
+type HashRingConfiguration struct {
```

Comment:
I think this should live here. But I can also see that it should be in the hashring package. Any thoughts?